### ISSUE TITLE
Power, or power spectral density of Periodogram?

### ISSUE THREAD

**Issue by KronosTheLate** [JuliaDSP/DSP.jl#480]
Currently, there is a function `power` that returns the field `power` of a Periodogram.

However, from what I can tell, the returned values are power spectral _densities_, which is of a different unit.
Am I correct in that understanding? If so, this should be reflected in the docstrings for `Periodogram`, `periodogram`, and 'power`.

**Comment by kcin96** [comment]
Yes, I agree it is power spectral density. I'll update the documentation.

**Comment by KronosTheLate** [comment]
Awesome, thanks for taking this issue seriously and fixing it for me ^_^
